### PR TITLE
[Zaraz] Pages reorganization

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Reorg content to better suit future features
- Added info on adding domains not proxied through Cloudflare
- Added info on how to add Zaraz manually

### CODE REVIEW

File: products/zaraz/src/content/advanced/debug-mode.md [hunk 1]
```diff
@@ -22,4 +22,9 @@ Zaraz offers a debug mode to troubleshoot the events and triggers systems. To ac
...
 Zaraz’s debug mode is now enabled. However, due to the temporary nature of the Console pane in web browsers, you need to activate the **Preserve Log** option for the Console pane. If you do not activate this option, you will lose the previous debug information whenever the Console refreshes.
 
-To activate the **Preserve Log** option in Google Chrome, click the **settings** cog > **Preserve logs**.
\ No newline at end of file
+To activate the **Preserve Log** option in Google Chrome:
+
+1. With the Developer tools opened, make sure you are in the **Console** pane.  
+1. Click the **Console settings** cog > **Preserve logs**.
+
+![Console pane](../static/console-settings.png)
```
Comment (deadlypants1973) [inline products/zaraz/src/content/advanced/debug-mode.md]:
```suggestion
To activate the **Preserve Log** option in Google Chrome:

1. Open **Developer Tools**.
1. Go to **Console**.
1. Select the **Console settings** cog > **Preserve logs**.

![Console pane](../static/console-settings.png)
```

Comment (deadlypants1973) [reply products/zaraz/src/content/advanced/debug-mode.md]: Just a suggestion for shorter sentences.

File: products/zaraz/src/content/advanced/domains-not-proxied.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+order: 7
+pcx-content-type: reference
+title: Domains not proxied by Cloudflare
+---
+
+# Use Zaraz on domains not proxied by Cloudflare
+
+You can load Zaraz on domains that are not proxied through Cloudflare. However, you will need to create a separate domain, or sub-domain, proxied by Cloudflare (also [known as orange-clouded](https://community.cloudflare.com/t/step-3-enabling-the-orange-cloud/52715) domains), and load the script from it:
```
Comment (deadlypants1973) [inline products/zaraz/src/content/advanced/domains-not-proxied.md]:
```suggestion
You can load Zaraz on domains that are not proxied through Cloudflare. However, you will need to create a separate domain, or subdomain, proxied by Cloudflare (also [known as orange-clouded](https://community.cloudflare.com/t/step-3-enabling-the-orange-cloud/52715) domains), and load the script from it:
```

File: products/zaraz/src/content/advanced/load-zaraz-manually.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+order: 6
+pcx-content-type: reference
+---
+
+# Load Zaraz manually
+
+If you turn off the [Auto-inject script](/reference/options) option, you will have manually include the Zaraz script in your HTML, immediately before the `</head>` tag closes. The path to your script would be `/cdn-cgi/zaraz/i.js`, on your domain. Your script tag should look like this:
```
Comment (deadlypants1973) [inline products/zaraz/src/content/advanced/load-zaraz-manually.md]: you will have to? 

or 

you will have manually included?

File: products/zaraz/src/content/reference/options.md [hunk 4]
```diff
@@ -8,37 +8,16 @@ title: Options
...
+Cloudflare Zaraz offers backwards compatibility with the `dataLayer` function found in tag management software, used to track events and other parameters. You can toggle this option off if you do not need it. Refer to [Data layer compatibility mode](/advanced/datalayer-compatibility) for more information.
 
 ## Single Page Application support
 
 When you toggle Single Page Application support off, the `pageview` trigger will only work when loading a new web page. When enabled, Zaraz's `pageview` trigger will work every time the URL changes on a single page application. This is also known as virtual page views.
 
 ## Auto-inject script
 
-This option automatically injects the script needed for Zaraz to work on your website. It is turned on by default.
+This option automatically injects the script needed for Zaraz to work on your website. It is turned on by default.
```
Comment (deadlypants1973) [inline products/zaraz/src/content/reference/options.md]:
```suggestion
This option automatically injects the script needed for Zaraz to work on your website. It is turned on by default.
```

File: products/zaraz/src/content/reference/triggers.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+<TableWrap>
+
+Rule type | Variable name | Match operation | Match string
+---       | ---           | ---             | ---
+_Match rule_ | `{{ client.__zarazTrack }}` |  _Contains_ | `purchase`
+
+</TableWrap>
+
+Refer to Rule types below for more information on the types of rules available, and [Zaraz event and system properties](/reference/properties-reference) for more information on the variables you can use to create triggers. 
+
```
Comment (deadlypants1973) [inline products/zaraz/src/content/reference/triggers.md:33]:
```suggestion
```

File: products/zaraz/src/content/reference/triggers.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+The rule type determines the kind of conditions Zaraz should listen for in your web page. Zaraz supports different rule types which allow you to create complex rules.
+
+<details>
+<summary>Match rule</summary>
+<div>
+
+Zaraz tracks the variable you input in **Variable name**. For a complete list of supported variables, refer to [Zaraz event and system properties](/reference/properties-reference).
+
+**Trigger example:**
```
Comment (deadlypants1973) [inline products/zaraz/src/content/reference/triggers.md]:
```suggestion
**Trigger example: Click listener**
```

File: products/zaraz/src/content/reference/triggers.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+<TableWrap>
+
+Rule type | Interval | Limit
+--- | --- | --- 
+_Timer_ | `50` | `2`
+
+</TableWrap>
+
+To improve the performance of the web page, you can limit a Timer trigger to a specific URL, by combining it with a Match rule. For example, to set up a timer on a specific page you can set up the following rules in a trigger:
```
Comment (deadlypants1973) [inline products/zaraz/src/content/reference/triggers.md]:
```suggestion
To improve the performance of a web page, you can limit a Timer trigger to a specific URL, by combining it with a Match rule. For example, to set up a timer on a specific page you can set up the following rules in a trigger:
```
Ignore this if you were talking about a specific webpage in the example, but as I read it, it didn't seem like that.

File: products/zaraz/src/content/web-api/zaraz-track.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+Common user actions you will probably be interested in tracking are successful sign-ups, calls-to-action clicks, and purchases. Common examples for other types of events are tracking the impressions of specific elements on a page, or loading a specific widget.
+
+To start tracking events, add `zaraz.track()` to the `<body>` element of your website. The following is an example for tracking purchases:
+
+```js
+zaraz.track("purchase", {value: "200", currency: "USD"})
+```
+
+Only the first parameter is required. This parameter should be a string specifying the name of the event you want to track. The second parameter is an optional free-form object that can be used to send an additional set of key-value properties. For example, if the `event` you are tracking is "purchase", this object can include `price`, `currency`, `transaction_id`, or `tax`. You choose what you want to track.
```
Comment (deadlypants1973) [inline products/zaraz/src/content/web-api/zaraz-track.md]:
```suggestion
Only the first parameter is required. This parameter should be a string specifying the name of the event you want to track. The second parameter is an optional free-form object that can be used to send an additional set of key-value properties. For example, if the `event` you are tracking is `purchase`, this object can include `price`, `currency`, `transaction_id`, or `tax`. You choose what you want to track.
```
Removing the " as per style guide. I usually either replace with ` if it's programmatic or just leave the " out entirely.

File: products/zaraz/src/content/web-api/zaraz-track.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+After using `zaraz.track()` in your website to track an action:
+
+1. Create a trigger that matches your [`zaraz.track()` call](/get-started/create-trigger), with `{{ client.__zarazTrack }}` in your trigger's variable name, and the parameter you are tracking in **Match string**. Following the above example, in **Match string** you would input `purchase`.
```
Comment (deadlypants1973) [inline products/zaraz/src/content/web-api/zaraz-track.md]:
```suggestion
1. Create a trigger that matches your [`zaraz.track()` call](/get-started/create-trigger), with `{{ client.__zarazTrack }}` in your trigger's variable name, and the parameter you are tracking in **Match string**. Following the above example, you would input `purchase` in **Match string**.
```

**Review by deadlypants1973 — APPROVED** [review]

File: products/zaraz/src/content/advanced/load-zaraz-manually.md (hunk 3 above)
Comment (marciocloudflare) [reply products/zaraz/src/content/advanced/load-zaraz-manually.md]: Will have to! I'll add it shortly.

**Pull request merged**